src/main/java/gov/hhs/cms/bluebutton/fhirstress/backend/RetrievePatientAndAllEobs.java

```diff
@@ -12,11 +12,11 @@
 import org.apache.jmeter.protocol.java.sampler.AbstractJavaSamplerClient;
 import org.apache.jmeter.protocol.java.sampler.JavaSamplerContext;
 import org.apache.jmeter.samplers.SampleResult;
-import org.hl7.fhir.dstu21.model.Bundle;
-import org.hl7.fhir.dstu21.model.Bundle.BundleEntryComponent;
-import org.hl7.fhir.dstu21.model.DomainResource;
-import org.hl7.fhir.dstu21.model.ExplanationOfBenefit;
-import org.hl7.fhir.dstu21.model.Patient;
+import org.hl7.fhir.dstu2016may.model.Bundle;
```

Comment:
I think these are the wrong model classes. Over in the transformer code ([bluebutton-server-app/gov.hhs.cms.bluebutton.server.app.stu3.providers](https://github.com/HHSIDEAlab/bluebutton-server/tree/master/bluebutton-server-app/src/main/java/gov/hhs/cms/bluebutton/server/app/stu3/providers)), we're using this package: `org.hl7.fhir.dstu3.model`.